env.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+OUTPUT_FILE=outputs.txt
+
+terraform init > /dev/null
+terraform apply --auto-approve > /dev/null
+terraform output >$OUTPUT_FILE
+
+function get_var() {
+  var_name=$1
+  grep -oP "(?<=${var_name} = ).+" $OUTPUT_FILE
+}
+
+OS_NETWORK_ID=$(get_var network_id)
+OS_VPC_ID=$(get_var vpc_id)
+OS_SUBNET_ID=$(get_var subnet_id)
+OS_IMAGE_ID=$(get_var image_id)
+
+echo "
+# To get environment variables for running terraform accepance tests, run
+# eval \$(./env.sh)
+#
+# Please note that you still have to provide auth variables:
+# OS_AUTH_URL, OS_TENANT_NAME, OS_USERNAME, OS_PASSWORD, OS_USER_DOMAIN_NAME, OS_ACCESS_KEY, OS_SECRET_KEY
```

Comment:
```
OS_PROJECT_NAME
OS_PROJECT_DOMAIN_NAME
```

Reply:
Please check `provider_test.go` for used variables

Reply:
OS_PROJECT_NAME exists

Reply:
`OS_TENANT_NAME` is enough:
![image](https://user-images.githubusercontent.com/8591561/98666232-c2ad4c80-235d-11eb-9804-a72904c6b9cd.png)